sections/Chapter 8 Sprint 6/ImplementingTheNewModels.tex

```diff
@@ -0,0 +1,87 @@
+\subsubsection*{Implementing the models}\label{implementing_new_WC_DB_models}
+Having established the quality of the design of the database structures, we will now implement them.
+When creating the new tables, we ensure that no loss of data occurs, simply by not dropping the old tables, and renaming them instead. 
+The tables can then be dropped later after the full implementations of the new design have been performed.
+The renaming of the tables can be seen in Code Snippet \ref{lst:RenamingOldModelsSQL}.
+
+\begin{lstlisting}[
+    label=lst:RenamingOldModelsSQL,
+    language=SQL,
+    caption=SQL script renaming the existing database structures,
+    showspaces=false,
+    numbers=left,
+    numberstyle=\tiny,
+    commentstyle=\color{gray},
+    escapechar=|
+ ]
+ ALTER TABLE "Article"
+    RENAME TO "Article_Redundant";
+ALTER TABLE "Term"
+    RENAME TO "Term_Redundant";
+\end{lstlisting}
+
+When implementing the new tables, we base them on the just-renamed tables from the old database design.
+By doing this, we ensure that all data from the old tables are correctly transferred to the new ones since each of the commands is a transaction (see section \ref{relational_databases}).
```

Comment:
```suggestion
By doing this, we ensure that all data from the old tables are correctly transferred to the new ones, since each of the commands is a transaction (see section \ref{relational_databases}).
```